modules/administration/pages/monitoring.adoc

```diff
@@ -217,7 +217,7 @@ image::monitoring_enable_services.png[scaledwidth=80%]
 
 . In the {susemgr} {webui}, open the details page of the system to be monitored, and navigate to the menu:Formulas[Prometheus Exporters] tab.
 . Check the [guimenu]``Enabled`` checkbox for both the Node and the Postgres Exporter.
-. In the [guimenu]``Postgres Exporter`` section, in the [guimenu]``Data Source Namer`` field, enter the path to your data source (for example, [systemitem]``postgresql://user:passwd@localhost:5432/database?sslmode=disable``).
+. In the [guimenu]``Postgres Exporter`` section, in the [guimenu]``Data Source Namer`` field, enter the the string [systemitem]``postgresql://USER:PASSWORD@localhost:5432/database?sslmode=disable``, replacing ``USER`` and ``PASSWORD`` with the values of the ``db_user`` and ``db_password`` from the ``/etc/rhn/rhn.conf`` file. 
```

Comment:
```suggestion
. In the [guimenu]``Postgres Exporter`` section, in the [guimenu]``Data Source Namer`` field, enter the string [systemitem]``postgresql://USER:PASSWORD@localhost:5432/database?sslmode=disable``, replacing ``USER`` and ``PASSWORD`` with the values of ``db_user`` and ``db_password`` from the ``/etc/rhn/rhn.conf`` file. 
```

Reply:
I agree we should always use PostgreSQL, which is the correct name